src/Glue.elm

```diff
@@ -2,11 +2,10 @@ module Glue exposing
     ( Glue
     , simple, poly, glue
     , init
-    , update, updateModel, updateWith, updateModelWith
+    , update, updateModel, updateWith, updateModelWith, trigger
```

Comment:
I'm using `elm-format 0.8.1` so if you don't want this change I can revert it.

Reply:
I'm also using this version. the reason why it moved it is that you haven't got `@docs trigger` when fmt ran for first time.